config/locales/en.yml

```diff
@@ -794,6 +794,11 @@ en:
             kind:
               not_found: "unavailable for this account type"
 
+        api_docs/service:
+          attributes:
+            base:
+              service_account_mismatch: 'The service must belong to the same account.'
```

Comment:
Well that does not really make sense in SaaS for example. Tenant are not supposed to know that we share the same database :) And they will surely think that account will refer to their developer which does not make sense and might lead them to more questions than answer.

Reply:
I pushed a different version https://github.com/3scale/porta/pull/62#discussion_r222252528 😄